recipes/ciocheck/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "ciocheck" %}
+{% set version = "0.1.1" %}
+{% set hash_type = "sha1" %}
```

Comment:
How do I get that from https://github.com/ContinuumIO/ciocheck/archive/v0.1.1.tar.gz ?